CMakeLists.txt

```
@@ -248,8 +248,9 @@ EndIf()
 
 find_package2(PUBLIC yaml-cpp)
 if(yaml-cpp_FOUND AND TARGET yaml-cpp AND YAML_CPP_INCLUDE_DIR)
+  get_filename_component(YAML_CPP_INCLUDE_TMP ${YAML_CPP_INCLUDE_DIR} ABSOLUTE)
```

```suggestion
  get_filename_component(YAML_CPP_INCLUDE_DIR "${YAML_CPP_INCLUDE_DIR}" ABSOLUTE)
```

I think it is safe to overwrite the existing `YAML_CPP_INCLUDE_DIR` variable here - no need for a new name.

---

Fine with me, the PR is editable by maintainers, so please adjust to your needs.

---

Mh, from the looks upstream seems to have [made this workaround obsolete](https://github.com/jbeder/yaml-cpp/commit/ab5f9259a4e67d3fe0e4832bd407a9e596e2d884) entirely after `0.6.2` (no release though). Eventually this will become:

```cmake
if(yaml-cpp_FOUND AND TARGET yaml-cpp AND YAML_CPP_INCLUDE_DIR AND yaml-cpp_VERSION VERSION_LESS 0.6.2)
  get_filename_component(YAML_CPP_INCLUDE_DIR "${YAML_CPP_INCLUDE_DIR}" ABSOLUTE)
  set_target_properties(yaml-cpp PROPERTIES INTERFACE_INCLUDE_DIRECTORIES ${YAML_CPP_INCLUDE_DIR})
endif()
```

Wanna go there immediately?

---

> so please adjust to your needs.

will do